src/components/ConnectToWallet/ConnectToWallet.tsx

```diff
@@ -77,12 +77,18 @@ const ConnectToWallet: React.FC<ConnectToWalletProps> = ({ onConnect }) => {
 				console.error(`Encounter error while connecting to ${wallet}.`);
 				console.error(e);
 			});
+			localStorage.setItem('chosenWallet', 'yes');
```

Comment:
Could this instead of being `yes` be the type of wallet that is being connected to? See my below comment